app/controllers/samples_controller.rb

```diff
@@ -223,36 +223,18 @@ def heatmap
         Viruses: ["Phage"]
       },
       metrics: [
+        "NT.r",
```

Comment:
You already have default value above. You can reorder these.

Reply:
ok